UnitTesting/UnitTests/EntityUnitTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using AutoMapper;
+using Domain;
+using Domain.Dto.BetableEntity;
+using Domain.Entities;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+using DataAccess;
+using Services;
+using Services.Interfaces;
+using UnitTesting.Scheduler;
+using Xunit.Abstractions;
+using UnitTesting.BetsUnitTests;
+using Domain.ErrorEntities;
+
+namespace UnitTesting.UnitTests
+{
+    public class EntityUnitTests
+    {
+        private readonly ITestOutputHelper _output;
+        private readonly IBetableEntityService _entityService;
+        private readonly IMapper _mapper;
+
+
+        public EntityUnitTests(ITestOutputHelper output)
+        {
+            _output = output;
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.AddProfile<MapperConfig>();
+            });
+            _mapper = config.CreateMapper();
+            var options = new DbContextOptionsBuilder<DBContext>()
+                                .UseInMemoryDatabase(Guid.NewGuid().ToString())
+                                .ConfigureWarnings(x => x.Ignore(InMemoryEventId.TransactionIgnoredWarning))
+                                .Options;
+            _entityService = new BetableEntityService(new DBContext(options), _mapper);
+        }
+
+        private async Task<BetableEntityDto> CreateEntity()
+        {
+            CreateBetableEntityDto createBetableEntityDto = new CreateBetableEntityDto { Name = $"BetableEntity {Guid.NewGuid()}" };
+            BetableEntityDto betableEntityDto = null;
+            Func<Task> act = async () =>
+            {
+                betableEntityDto = await _entityService.CreateAsync(createBetableEntityDto);
+            };
+            await act.Should().NotThrowAsync();
```

Comment:
no need for this., you can just invoke the method - we don't expect it to fail, do we ?